circle.yml

```diff
@@ -1,14 +1,9 @@
 ---
-deployment:
-  production:
-    branch: master
-    commands:
-      - "echo \"machine api.heroku.com login jake.champion@ft.com password ${HEROKU_AUTH_TOKEN}\" >> ${HOME}/.netrc;"
```

Comment:
This has been pulled out as it does not work as blocked by https://github.com/heroku/heroku-docker/issues/138